contracts/LP/L1LiquidityPool.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: MIT
+// @unsupported: ovm
+pragma solidity >0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "./interfaces/iL2LiquidityPool.sol";
+
+/* Library Imports */
+import "omgx_contracts/build/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol";
+
+/* External Imports */
+import '@openzeppelin/contracts/math/SafeMath.sol';
+import '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+/**
+ * @dev An L1 LiquidityPool implementation
+ */
+contract L1LiquidityPool is OVM_CrossDomainEnabled, Ownable {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+
+    uint256 constant internal SAFE_GAS_STIPEND = 2300;
+
+    /*************
+     * Variables *
+     *************/
+     
+    // this is to stop attacks where caller specifies l2contractaddress
+    // also acts as a whitelist
+    mapping(address => address) l2ContractAddress;
+    
+    address l2LiquidityPoolAddress;
+
+    /********************
+     *    Constructor   *
+     ********************/
+    constructor(
+        address _l2LiquidityPoolAddress,
+        address _l1messenger,
+        address _l2ETHAddress
+    )
+        OVM_CrossDomainEnabled(_l1messenger)
+    {
+        l2LiquidityPoolAddress = _l2LiquidityPoolAddress;
+        l2ContractAddress[address(0)] = _l2ETHAddress;
+    }
+
+    /********************
+     *       Events     *
+     ********************/
+
+    event clientDepositL1_EVENT(
+        address sender,
+        uint256 amount,
+        address l1TokenAddress,
+        address l2TokenAddress
+    );
+
+    event clientPayL1_EVENT(
+        address sender,
+        uint256 amount,
+        address l1TokenAddress,
+        address l2TokenAddress
+    );
```

Comment:
can we change the namings of these events? I don't see a particular reason to include capital `EVENT`.